drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -149,6 +149,97 @@ local child_device_profile_overrides = {
 
 local detect_matter_thing
 
+local CUMULATIVE_REPORTS_NOT_SUPPORTED = "__cumulative_reports_not_supported"
+local FIRST_EXPORT_REPORT_TIMESTAMP = "__first_export_report_timestamp"
+local EXPORT_POLL_TIMER_IS_SET = "__export_poll_timer_is_set"
+local EXPORT_REPORT_TIMEOUT = "__export_report_timeout"
+local TOTAL_EXPORTED_ENERGY = "__total_exported_energy"
+local LAST_EXPORTED_REPORT_TIMESTAMP = "__last_exported_report_timestamp"
+local RECURRING_EXPORT_REPORT_POLL_TIMER = "__recurring_export_report_poll_timer"
+local MINIMUM_ST_ENERGY_REPORT_INTERVAL = (15 * 60) -- 15 minutes, reported in seconds
+local SUBSCIPTION_REPORT_OCCURRED = "__subscription_report_occurred"
+
+local embedded_cluster_utils = require "embedded-cluster-utils"
+
+-- Include driver-side definitions when lua libs api version is < 11
+local version = require "version"
+if version.api < 11 then
+  clusters.ElectricalEnergyMeasurement = require "ElectricalEnergyMeasurement"
+  clusters.ElectricalPowerMeasurement = require "ElectricalPowerMeasurement"
+  clusters.ValveConfigurationAndControl = require "ValveConfigurationAndControl"
+end
+
+-- Return an ISO-8061 timestamp in UTC
+local function iso8061Timestamp(time)
+  return os.date("!%Y-%m-%dT%TZ", time)
```

Comment:
I'd probably change to  `!%Y-%m-%dT%H:%M:%SZ` to be more specific and match the other implementation of this function in this PR

Reply:
sounds good :+1: